Update basic/bubble_sort (#13)
OI-wiki/i18n: docs/basic/bubble-sort.md

> Bubble sorting is a stable sorting algorithm.

Reviewer: https://en.m.wikipedia.org/wiki/Category:Stable_sorts
稳定排序容易理解错，建议加个 wiki 链（提升原文

Taking ascending order as an example, the bubble sort checks two adjacent elements each time. If the previous element is larger than the following element, the adjacent two elements are swapped. When no adjacent elements need to be exchanged, the sorting is complete.

> After $i$ scans, the $i$ item at the end of the sequence must be the largest $i$ item, so you need to scan at most $n-1$ through the array to complete the sorting.

Reviewer: ```suggestion
After $i$ scans, the $i$-th item at the end of the sequence must be among the largest $i$ items, so at most $n-1$ scans through the array is needed to complete the sorting.
```

> When the sequence is completely ordered, the algorithm only needs to traverse the array once, without performing any swap operation, and the time complexity is $O(n)$ . In the worst case, bubble sorting needs to perform $\frac{(n-1)n}{2}$ exchange operations, and the time complexity is $O(n^2)$ . In the average case, the time complexity of bubble sort is also $O(n^2)$ .

Reviewer: in average 即可

\end{array}
$$

C++ code:

```cpp

> // Assuming that the size of the array is n+1, the bubble sort usually starts from the index 1

Reviewer: ```suggestion
// Assuming that the size of the array is n+1, and the index starts from 1
```
